src/legacy/core_plugins/tests_bundle/tests_entry_template.js

```diff
@@ -94,7 +94,8 @@ const coreSystem = new CoreSystem({
       uiSettings: {
         defaults: ${JSON.stringify(defaultUiSettings, null, 2).split('\n').join('\n    ')},
         user: {}
-      }
+      },
+      nav: []
```

Comment:
This code isn't linted because it's actually a big template string, but I'll add go ahead and add the trailing comma.

Reply:
Not sure why this isn't linted, but I think this should have a trailing comma.